src/main/java/com/quizter/entity/PasswordResetToken.java

```diff
@@ -0,0 +1,34 @@
+package com.quizter.entity;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import java.util.Date;
+
+@Builder
+@Entity
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class PasswordResetToken {
+
+    static final int EXPIRATION = 60 * 24;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    Long id;
+
+    String token;
+
+    @OneToOne(targetEntity = User.class, fetch = FetchType.EAGER)
+    @JoinColumn(nullable = false, name = "user_id")
+    User user;
+
+    Date expiryDate;
```

Comment:
Don't use java.util.Date. Use java.time.Instant. Date is legacy, most methods in it are deprecated

Reply:
It's mine either, I'll fix it in my branch